### ISSUE TITLE
Top 3(-5) lists

### ISSUE THREAD

**Issue by ANRTheBigBoy** [ACG8/Lucid#264]
Post your top 3 for each category. If you feel it's close you can go up to a top 5, but only do so if it's really a dead-heat for the last slot.

Fun cards: "I get excited when I see this in my first pack. I want to take this and play with it and when I do it's always fun."

Powerful cards: "If all you told me about someone's deck is that it had this card in it, I would be inclined to say it's a strong deck." or "If I was playing a game of lucid for $100, this is the first card I would want in my deck."

Frustrating/Hated cards: "When another player plays this card, I feel frustrated or annoyed. I have less fun when this card is in the game."

Boring/Ignored cards: "My eyes skip over this card when I see it in a pack. It may or may not be a good card, but I have no interest in playing with it for whatever reason."

**Comment by millerj-tc** [comment]
Let's all post that we're ready before we post the actual list so that we don't influence each other's choices.

**Comment by ACG8** [comment]
Ready

**Comment by ACG8** [comment]
Fun Cards

- The Cleanser
Quintessential scoring card for handless deck, which I love using. Straightforward and easy to trigger for you, but painful for your opponents.0
- Subliminal Surveyor
Random prepares at no extra cost are fun. Mitigates painfulness of hitting a dive.
- Desert Basilisk
Fairly easy double-access ability that can be used at pretty much any point of the game.

Powerful Cards
- Silhouette of the Guardian
Allows you to stop playing dives, opening up space for more dreams and lucids
- Sly Fox Planet
Either ability alone is solid; consistent at getting you will and insight
- Eyyyyyy
Very powerful scoring ability, and difficult for hand draw to steal

Hated Cards
I don't really hate any cards very much. These ones aren't that fun to play against, but I don't have especially strong feelings against them.
- Brain Drain
- Adelia's Whim
- Cyberloxo

Ignored Cards
- Dream Vacation
We nerfed it when it needed a buff. Pretty much worthless, compared to its companions. Also odd that it is the only ___-less card that doesn't give insight baseline.
- The Red Wizard
Baseline ability is meh. Sacrifice ability is overpriced.
- Hope of Raziel
Basically just a random dream from your mind. I pass over Down the Rabbit Hole for the same reason. Could be good if there is one dream you want to hit repeatedly, but you can accomplish the same thing with card draw without having to give up a slot.
